Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -501,19 +513,15 @@ extension ProjectDescription.Platform {
         )
         let validPlatforms = configuredPlatforms.intersection(packagePlatforms)
```

Comment:
This should no longer be required, we can just generate targets for all configured platforms

Reply:
Removed this part of the code